instrumentation/netty/netty-4.0/javaagent/src/test/groovy/Netty40ClientTest.groovy

```diff
@@ -139,9 +139,4 @@ class Netty40ClientTest extends HttpClientTest<DefaultFullHttpRequest> implement
   boolean testHttps() {
     false
   }
-
-  @Override
-  boolean testReadTimeout() {
```

Comment:
Now that we've switched over to using `NetClientAttributesExtractor` in netty instrumentations the `net.peer.*` attributes are no longer captured (`channel.remoteAddress()` returns null in case of timeout) and this test case fails. 
There are no more tests in any instrumentations that enable it - we should verify how other clients behave and, if the lack of `net.*` attributes is a common thing, change the test scenario to reflect that. I'll create a separate issue for this once this PR gets merged.

Reply:
> btw, is the problem that `Channel.remoteAddress()` returns null in `onEnd` (while it returns non-null earlier in `onStart`)?

Yep, it's exactly that. I believe that we can do 2 things to fix that:
1. change the assertions in the timeout test so that they don't expect `net.*` attributes - this has a downside of losing some attributes that, as Lauri said, might be incredibly useful on a failing request span;
2. always capture `net.*` attributes on start - which would probably force us to re-think the net attributes extractors naming; it is also worth noticing that there were some HTTP clients that reported `net*` attributes on start, so it could be worth implementing anyway.

I think I'm leaning towards option 2, and just simply reusing the server net attributes extractor in this PR.